flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

Comment:
```suggestion
  "only-arches": ["x86_64"]
```

Reply:
This change has been implemented and a new build without i386 libraries is now in the manifest.